docs/device_report.md

```diff
@@ -1,33 +1,43 @@
 # DAQ scan report for device 9a02571e8f01
-Started %% 2019-05-07 04:06:49+00:00
-
-|  Role  |      Name              |
-|--------|------------------------|
-|Operator| *** Operator Name *** |
-|Reviewer| *** Reviewer Name *** |
-|Approver| *** Approver Name *** |
-|--------|------------------------|
-| Test report date | 2019-05-07T04:06:49.499Z |
-| DAQ version      | 0.9.6 |
-
-## Device Identification
-
-| Device        | Entry              |
-|---------------|--------------------|
-| Name          |  |
-| GUID          |  |
-| MAC addr      | 9a:02:57:1e:8f:01 |
-| Hostname      |  |
-| Type          |  |
-| Make          |  |
-| Model         |  |
-| Serial Number |  |
-| Version       |  |
+Started
```

Comment:
This doesn't look right -- the %% is missing.  How did you get this file here?  It should be a literal copy from the corresponding test output directory... ?

Reply:
Yes, sorry. I was lazy to copy the wrong file. I'll fix it in the next commits.